README.md

```diff
@@ -96,19 +127,20 @@ ckan_api_load_single_step
 ```
 
 3. Make sure you have these environment variables properly set up:
+
 ```bash
 export LC_ALL=en_US.UTF-8
 export LANG=en_US.UTF-8
 export OBJC_DISABLE_INITIALIZE_FORK_SAFETY=YES
 ```
 
-4. Run the airflow webserver: `airflow webserver`
+4. Run the Airflow webserver: `airflow webserver`
 
-5. Run the airflow scheduler: `airflow scheduler`. Make sure the environment variables from (3) are set up.
+5. Run the Airflow scheduler: `airflow scheduler`. Make sure the environment variables from (3) are set up.
 
 6. Access the Airflow UI (`http://localhost:8080/`). You should see the DAG `ckan_api_load_single_step` listed.
 
-7. Activate the DAG by hitting on the **On** button on the interface.
+7. Activate the DAG by hitting on the **Off** button on the interface.
```

Comment:
The DAGs should be in the **Off** position by default. (At least mine show like that.)